app/javascript/components/explore/CellFilteringPanel.jsx

```diff
@@ -31,6 +30,244 @@ export function CellFilteringPanelHeader({
   )
 }
 
+/**
+ * Very brief notes on terms in SCP metadata convention, derived from:
+ * https://singlecell.zendesk.com/hc/en-us/articles/360060609852-Required-Metadata
+ */
+const conventionalMetadataGlossary = {
+  'biosample_id': 'Unique identifier for each sample in the study',
+  'donor_id': 'Unique identifier for each biosample donor in the study',
+  'species__ontology_label': 'Taxon name, from NCBITaxon',
+  'disease__ontology_label': 'Disease name, from Mondo or PATO',
+  'organ__ontology_label': 'Organ name, from Uberon',
+  'library_preparation_protocol__ontology_label': 'From EFO',
+  'sex': 'One of "female", "male", "mixed", or "unknown"',
+  'cell_type__ontology_label': 'From Cell Ontology',
+  'ethnicity__ontology_label': 'From Human Ancestry Ontology'
+}
+
+/** Determine if annotation is conventional from its raw name */
+function getIsConventionalAnnotation(rawName) {
+  return (
+    rawName.includes('__ontology_label') ||
+    ['donor_id', 'biosample_id'].includes(rawName)
+  )
+}
+
+/** Convert e.g. "cell_type__ontology_label" to "Cell type" */
+function parseAnnotationName(annotationIdentifier) {
+  const rawName = annotationIdentifier.split('--')[0]
+  const sansOntologyName = rawName.replace('__ontology_label', '')
+  const sentenceCased = sansOntologyName[0].toUpperCase() + sansOntologyName.slice(1)
+  const spaced = sentenceCased.replace(/_/g, ' ')
+  let upId = spaced
+  if (spaced.slice(-3) === ' id') {
+    // e.g. Donor id -> Donor ID
+    upId = upId.slice(0, -3) + upId.slice(-3).toUpperCase()
+  }
+  let ynLess = upId
+  if (upId.slice('-3').toUpperCase() === ' YN') {
```

Comment:
For reference, here's all unique matching annotations in production that would encounter this rule:
```
 "infant_sick_YN",                                                                                          
 "weaning_YN",                                                                                              
 "mastisis_YN",                                                                                             
 "breast_soreness_YN",                                                                                      
 "directly_breastfeeding_YN",                                                                               
 "any_formula_YN",                                                                                          
 "mother_medications_YN",                                                                                   
 "reported_menstruating_YN",                                                                                
 "maternal_medical_event_YN",                                                                               
 "hormonal_birthcontrol_YN",                                                                                
 "daycare_YN",                                                                                              
 "vaccines_reported_YN",                                                                                    
 "solid_foods_YN",                                                                                          
 "reported_infant_medical_events_YN",                                                                       
 "labor_induced_YN"
```

I checked, and there's no analog for `true/false` that I can find, though that may be something we could consider moving forward.